docs/DeveloperGuide.md

```diff
@@ -264,13 +264,25 @@ The diagram highlights the four main components of FitBook, highlighted in their
 > The above sequence diagram also applies to the removal of a note from an existing client when no input string or prefix is entered for the `note` command. (i.e. `note 1`, or `note 1 nt/`).
 
 #### Weight tracking feature
-The weight tracking feature allows users to keep track of past weight measurements of a client.
+The weight tracking feature allows users to keep track of past weight measurements of a client. Since a client is a person, we have set a more-than-reasonable limit of 5000kg for `weight`.
```

Comment:
I think we should compile all the valid parameters together in a table and just reference that table when needed.